apps/vr-tests-react-components/src/stories/Table.stories.tsx

```diff
@@ -634,6 +684,85 @@ const Truncate: React.FC<SharedVrTestArgs & { truncate?: boolean }> = ({ noNativ
   </Table>
 );
 
+const ResizableColumns: React.FC<SharedVrTestArgs & { scrollToEnd?: boolean }> = ({
+  noNativeElements,
+  scrollToEnd,
+}) => {
+  const [columnSizingOptions] = React.useState({
+    file: {
+      idealWidth: 300,
+      minWidth: 300,
+    },
+  });
+
+  const { columnSizing_unstable: columnSizing, tableRef } = useTableFeatures(
+    {
+      columns: columnsDef,
+      items,
+    },
+    [
+      useTableColumnSizing_unstable({
+        columnSizingOptions,
+      }),
+    ],
+  );
+  return (
+    <div style={{ width: '500px', overflow: 'auto' }}>
```

Comment:
Since headless browsers don't display scrollbars by default it might be nice to add a dashed border around the overflow container

Reply:
sounds good :)